templates/zerver/integrations/errbot.md

```diff
@@ -25,23 +28,30 @@ Run your favorite chatbot in Zulip!
         BOT_LOG_FILE = r'<path/to/your/errbot/logfile.log>'
         BOT_LOG_LEVEL = logging.INFO
 
-        BOT_IDENTITY = {  # Fill this with the corresponding values in your bot's `.zuliprc`
+        BOT_IDENTITY = {  # Use the corresponding values in your bot's `.zuliprc`
           'email': '<err-bot@your.zulip.server>',
           'key': '<abcdefghijklmnopqrstuvwxyz123456>',
           'site': '<http://your.zulip.server>'
         }
         BOT_ADMINS = ('<your@email.address',)
         CHATROOM_PRESENCE = ()
-        BOT_PREFIX = '<@**err-bot**>'  # Providing errbot's full name in Zulip lets it respond to @-mentions.
+        BOT_PREFIX = '<@**err-bot**>'  # Providing your errbot's full name
```

Comment:
The comment wraps to the next line anyways, so split it to comment out the next line as well.

Reply:
Unless we plan on updating the errbot repository, I'd keep this example code block to be the same as the one there. Even though it doesn't match our documentation line-wrapping convention.